packages/ui/src/components/Alert/Alert.stories.tsx

```diff
@@ -1,75 +1,73 @@
 import React from 'react'
 
-import { Alert } from './../../../'
-
-// import ThemeProvider from '../../theme/themeContext'
+import Alert, { AlertProps } from './Alert'
 
 export default {
   title: 'Displays/Alerts',
   component: Alert,
 }
 
```

Comment:
can you refactor these to be kitchen-sink style. These don't need to be separate stories

Reply:
leaving for now